www/src/data/sidebars/doc-links.yaml

```diff
@@ -1,9 +1,8 @@
-- title: Quick Start
+- title: Documentation
+  link: /docs/
   items:
-    - title: Getting Started
-      link: /docs/
-    - title: Starter Library
-      link: /starters/
```

Comment:
Do we have Starter Library linked anywhere else?

Reply:
Planning to merge the ecosystem homepage update today -- the .org redesign will take care of linking the starter library, plugin library, etc.

^That piece of work doesn't account for changing the nav yet though. So perhaps should leave this here, until it's finished.

Another option would be to have another category section in docs "Ecosystem", that mirrors the new Ecosystem landing page -- link out to plugin library, starter library, etc etc.

<img width="1418" alt="screen shot 2018-11-13 at 9 45 48 am" src="https://user-images.githubusercontent.com/3461087/48424777-f7fc1100-e728-11e8-8fcf-68d3f13dc39e.png">

Actually this would be great too, because we could reference what they are, top-level, on the new docs landing page.

@shannonbux what do you think?